### ISSUE TITLE
SyntaxError on the client side in python 2 

### ISSUE THREAD

**Issue by HYPJUDY** [hanxiao/bert-as-service#220]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [ ] Are you running the latest `bert-as-service`?
* [ ] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [ ] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#speech_balloon-faq)?
* [ ] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**System information**

> Some of this information can be collected via [this script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh).

- OS Platform and Distribution: Linux Ubuntu 16.04:
- TensorFlow installed from (source or binary):
- TensorFlow version:1.4.1
- Python version:python 2.7.15 :: Anaconda, Inc
- `bert-as-service` version: 1.7.8
- GPU model and memory: GeForce GTX 1080, 12G
- CPU model and memory:

---

### Description
I have successfully called the client by python3 but I got problem when calling the client by python2.

I'm using this command to start the server:

```bash
bert-serving-start -model_dir uncased_L-12_H-768_A-12/ -num_worker=2 -pooling_strategy NONE
```

and calling the server by **python2** via:
```bash
$ python2
>>> from bert_serving.client import BertClient
Traceback (most recent call last):
    File "<stdin>", line 1, in <module>
    File "/home/user/.local/lib/python2.7/site-packages/bert_serving/client/__init__.py", line 186
        raise t_e from _e
SyntaxError: invalid syntax
```

Then the above issue shows up. 

I want to use BertClient as a feature extractor in a pytorch project with python2. Please help me with this. Thanks!

**Comment by hanxiao** [comment]
No need to change anything. It supports py2 out of the box. See:

https://github.com/hanxiao/bert-as-service/blob/b79a832b17e78060575fada6aa78dcd3083a9456/client/bert_serving/client/__init__.py#L21-L31

Not sure what's going on there, it works on my test env.
```bash
Python 2.7.14 (default, Sep 25 2017, 09:54:19)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from bert_serving.client import BertClient
>>>
```

**Comment by HYPJUDY** [comment]
Thanks @xpact and @hanxiao . I modified L186 from `raise t_e from _e` to `raise t_e`. Then I run `from bert_serving.client import BertClient` without error.
https://github.com/hanxiao/bert-as-service/blob/b79a832b17e78060575fada6aa78dcd3083a9456/client/bert_serving/client/__init__.py#L183-L186
I print `_py2` and the output is `True` but I don't know why the code will go into L186 (which means `_py2` is `False`).
``` bash
>>> sys.version_info
sys.version_info(major=2, minor=7, micro=15, releaselevel='final', serial=0)
>>> sys.version_info >= (3, 0)
False
```
Then I call the client via the following codes without error:
```
from bert_serving.client import BertClient
bc = BertClient()
bc.encode(['First do it', 'then do it right', 'then do it better'])
```
but I encoutered error with the following codes (from [README](https://github.com/hanxiao/bert-as-service#using-your-own-tokenizer)):
```
>>> texts = ['hello world!', 'good day']
>>> texts2 = [s.split() for s in texts]
>>> vecs = bc.encode(texts2, is_tokenized=True)
Traceback (most recent call last):
    File "<stdin>", line 1, in <module>
    File "/home/user/.local/lib/python2.7/site-packages/bert_serving/client/__init__.py", line 176, in arg_wrapper
         return func(self, *args, **kwargs
    File "/home/user/.local/lib/python2.7/site-packages/bert_serving/client/__init__.py", line 254, in encode
         texts = _unicode(texts)
    File "/home/user/.local/lib/python2.7/site-packages/bert_serving/client/__init__.py", line 31, in <lambda>
        _unicode = lambda x: [BertClient._force_to_unicode(y) for y in x]
    File "/home/user/.local/lib/python2.7/site-packages/bert_serving/client/__init__.py", line 361, in _force_to_unicode
        return text if isinstance(text, unicode) else text.decode('utf-8')
AttributeError: 'list'object has no attribute 'decode'
```

**Comment by hanxiao** [comment]
A second check on python2, there was indeed some bugs. Thanks @HYPJUDY and @xpact for pointing it out.

Fixed in #222 and available since 1.7.9, please do
```bash
pip install -U bert-serving-server bert-serving-client
```
for the update.
